src/test/kotlin/build/buf/gradle/AbstractBuildTest.kt

```diff
@@ -38,6 +40,13 @@ abstract class AbstractBuildTest : AbstractBufIntegrationTest() {
         assertThat(result.output).contains("found 0")
     }
 
+    @ParameterizedTest
+    @EnumSource(AvailablePublicationFileExtension::class)
```

Comment:
Is the class required as an argument here?

Reply:
Chnage it to use `@MethodSource`.